templates/_helpers.yaml

```diff
@@ -78,12 +78,16 @@
   {{ end }}
 {{- end }}
 
+{{ define "deployments_host" -}}
+  {{ printf "deployments.%s" .Values.ingress.baseDomain }}
+{{- end }}
+
 {{ define "airflow_url" -}}
-{{ .Release.Name }}-airflow.{{ .Values.ingress.baseDomain }}
+  {{ printf "deployments.%s/%s/airflow" .Values.ingress.baseDomain .Release.Name }}
```

Comment:
Can you call one macro from another? If so using `deployments_host` here would be good.